test/functional/lua/secure_spec.lua

```diff
@@ -168,4 +168,111 @@ describe('vim.secure', function()
       eq(false, curbufmeths.get_option('modifiable'))
     end)
   end)
+
+  describe('trust()', function()
+    local xstate = 'Xstate'
+
+    setup(function()
+      helpers.mkdir_p(xstate .. pathsep .. (is_os('win') and 'nvim-data' or 'nvim'))
+    end)
+
+    teardown(function()
+      helpers.rmdir(xstate)
+    end)
+
+    before_each(function()
+      helpers.write_file('test_file', 'test')
+    end)
+
+    after_each(function()
+      os.remove('test_file')
+    end)
+
+    it('returns error when passing both path and bufnr', function()
+      eq({false, 'path and bufnr are mutually exclusive'},
+        exec_lua([[return {vim.secure.trust({action='deny', bufnr=0, path='test_file'})}]]))
```

Comment:
Update this test for the new input validation. You can use the `pcall_err` helper function to test for an error.

Reply:
Whoops, sorry for missing that, was committing in haste :)
Done in 4da67a8f9d0326f4823d23e9ff2dc0122f00fb0f